src/DynamoApplications/StartupUtils.cs

```diff
@@ -194,8 +194,7 @@ public static void PreloadShapeManager(ref string geometryFactoryPath, ref strin
 
             var versions = new[]
             {
-                new Version(228,0,0),
-                new Version(227,0,0),
+                new Version(228,6,0),
```

Comment:
I'm now thinking that some integrators may not upgrade to ASM 228.6, and may stick with 228.5 so I was wondering if this is going to be a minimum required version, it might be safer to have 228.5 instead. I don't believe there should be any code changes required other than testing libg with ASM 228.5. Sorry for the back and forth on this.

Reply:
no worries, I can move all of these to 228.5 and then test with both 228.5 and .6 to see if basic sanity checks work.